.cirrus/Dockerfile

```diff
@@ -95,6 +97,19 @@ RUN set -eux; \
     mv ./kubeconform /usr/bin/kubeconform ; \
     rm kubeconform-linux-amd64.tar.gz
 
+RUN set -eux; \
+    curl -LO https://github.com/redhat-certification/chart-verifier/releases/download/${CHART_VERIFIER_VERSION}/chart-verifier-${CHART_VERIFIER_VERSION}.tgz ; \
+    echo "${CHART_VERIFIER_CHECKSUM}  chart-verifier-${CHART_VERIFIER_VERSION}.tgz" | sha256sum -c ; \
+    tar -xf chart-verifier-${CHART_VERIFIER_VERSION}.tgz ; \
+    chmod +x ./chart-verifier ; \
+    mv ./chart-verifier /usr/bin/chart-verifier ; \
+    rm chart-verifier-${CHART_VERIFIER_VERSION}.tgz
+
+RUN curl -LO https://github.com/derailed/k9s/releases/download/v0.32.5/k9s_Linux_amd64.tar.gz && \
```

Comment:
Is it still needed after debug/development phase ?

Reply:
Ops! Good catch! I forgot this.